src/problems/MetalAdvectionProblem/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+if (AMReX_SPACEDIM EQUAL 3)
+      add_executable(test_sne test_sne.cpp ${QuokkaObjSources})
+    if(AMReX_GPU_BACKEND MATCHES "CUDA")
+        setup_target_for_cuda_compilation(test_sne)
+    endif()
+
```

Comment:
Can you add a CTest `add_test(...)` command here? You can look at the CMakeLists.txt in the other problems for examples.